README.md

```diff
@@ -526,15 +528,15 @@ This buildpack combines several buildpacks, specified in [`.buildpacks`](.buildp
    * installs `node`, puts on the `$PATH`
    * version specified in [`package.json`, `engines.node`](https://devcenter.heroku.com/articles/nodejs-support#specifying-a-node-js-version)
    * `node_modules/` cached between deployments
+   * production build for create-react-app
+     * [executes the npm package's build script](https://devcenter.heroku.com/changelog-items/1557); create-react-app default is `react-scripts build`
```

Comment:
You may want to expand on this behavior since it will now the same as the default Heroku behavior, or link to [this section of the DevCenter](https://devcenter.heroku.com/articles/nodejs-support#customizing-the-build-process) which will be updated when the change is rolled out.